Sources/VariadicsGenerator/VariadicsGenerator.swift

```diff
@@ -14,7 +14,7 @@
 import ArgumentParser
 #if os(macOS)
```

Comment:
Seems like this should be `#if os(Darwin)`, even though we're not running it on any non-mac Darwin targets at present.

Reply:
(obviously, if we're gonna fix this, fix it on main first)

Reply:
@natecook1000  thoughts?